### ISSUE TITLE
get_NetworkTimeout does not have an implementation

### ISSUE THREAD

**Issue by ThomasMPorter** [Azure/azure-storage-net-data-movement#223]
### Which service(blob, file) does this issue concern? 
file
 
### Which version of the SDK was used?
1.3.0

### On which platform were you using? (.Net Framework version or .Net Core version, and OS version)
.Net Core 3.1

### How can the problem be reproduced? It'd be better if the code caused the problem can be shared.
Call to
`Microsoft.Azure.Storage.DataMovement.TransferManager.DownloadDirectoryAsync(CloudBlobDirectory sourceBlobDir, String destPath, DownloadDirectoryOptions options, DirectoryTransferContext context, CancellationToken cancellationToken)`

 
### What problem was encountered?

Exception when accessing  
`Method 'get_NetworkTimeout' in type 'Microsoft.Azure.Storage.File.FileRequestOptions' from assembly 'Microsoft.Azure.Storage.File, Version=11.1.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' does not have an implementation.`
 
### Have you found a mitigation/solution?
Using the 1.1.0 version of the SDK or also including Microsoft.Azure.Storage.File 11.1.7 in our package references

Fixes to **NetworkTimeout** have been addressed in Microsoft.Azure.Storage 11.1.7: <https://github.com/Azure/azure-storage-net/issues/999>

**Comment by blueww** [comment]
@ThomasMPorter 

Thanks for reporting this issue!
This is an issue in Microsoft.Azure.Storage.File code instead of DMlib code.

I see you have already work around it by upgrade Microsoft.Azure.Storage.File 11.1.7, since the issue is already resolved in 11.1.7.

The next DMlib release will base on Microsoft.Azure.Storage.File 11.1.7 or later version, which should not have this problem.

Do you mind I close this issue, as it's not in DMlib code, already has workaround, and definitely will be resolved in the next release.

**Comment by ThomasMPorter** [comment]
Sure, we can close, all our testing after the upgrade of File to 11.1.7 pass. We’ll watch out for the next version of DMlib. Thanks!

**Comment by sanderaernouts** [comment]
For anyone else having this issue but with downloading blobs instead, make sure you update `Microsoft.Azure.Storage.Blob` to `11.1.7` as well. `Microsoft.Azure.Storage.Blob.Blob.RequestOptions` has the same issue with `get_NetworkTimeout` and it depends on how you invoke the `TransferManager` in this library whether `Microsoft.Azure.Storage.File.FileRequestOptions` or `Microsoft.Azure.Storage.Blob.Blob.RequestOptions` are configured.

**TL;DR;**
Just to be on the safe side make sure your project references version `11.1.7` of these packages
- Microsoft.Azure.Storage.File
- Microsoft.Azure.Storage.Blob
